supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiConfigurationAttributes.scala

```diff
@@ -35,6 +35,7 @@ case class PipelinesApiConfigurationAttributes(project: String,
                                                qps: Int Refined Positive,
                                                cacheHitDuplicationStrategy: PipelinesCacheHitDuplicationStrategy,
                                                requestWorkers: Int Refined Positive,
+                                               pipelineTimeout: String,
```

Comment:
I think I would prefer the case class to be as type-safe and "always valid" as possible. In this case I'd have it hold an `Option[FiniteDuration]` rather than a "`String` which we hope is the right format".

I would then put the logic to stringify the duration all the way down in the `GenomicsFactory`

Reply:
I agree, that makes more sense than a String. Done!